### PR TITLE
IA-1803: Consolidating OAIs across deployments in the SBA.gov infrastructure

### PR DESCRIPTION
**Pull request by bushong1**

### CODE REVIEW

**Review by jsmigovsky — APPROVED** [review]

File: solution/infrastructure/config/infra/cloudformation.yml [hunk 1]
```diff
@@ -155,13 +86,14 @@ Resources:
            MinimumProtocolVersion: TLSv1.1_2016
            SslSupportMethod: sni-only
         PriceClass: PriceClass_100
-        WebACLId: !GetAtt rWAFWebACL.Arn
+        #WebACLId: #TODO: Use global waf from parameter_store when available
```
Comment (kscoulter) [inline solution/infrastructure/config/infra/cloudformation.yml:89]: How are you making sure this TODO will be accomplished? Is there a time frame on getting the global waf?

Comment (bushong1) [reply solution/infrastructure/config/infra/cloudformation.yml]: @kscoulter Thanks for the encouragement :-) It's taken care of now.

File: solution/infrastructure/config/infra/cloudformation.yml [hunk 2]
```diff
@@ -30,81 +30,13 @@ Resources:
             - 's3:GetObject'
           Effect: Allow
           Principal:
-            AWS: !Join ['',['arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity ', !Ref 'CloudFrontOAI']]
+            AWS:
+            - ${self:custom.settings.oai.iam_arn}
+            #TODO: delete below and de-array above
```
Comment (kscoulter) [inline solution/infrastructure/config/infra/cloudformation.yml:35]: Is this still a valid comment?

**Review by kscoulter — DISMISSED** [review]
Thanks for the PR :)

File: solution/infrastructure/config/infra/cloudformation.yml (hunk 2 above)
Comment (bushong1) [reply solution/infrastructure/config/infra/cloudformation.yml:35]: Yes.  In short: we need to apply these changes to allow _both_ OAI's to read the bucket, then apply changes on the sba-gov-inf side to _use_ the new OAI, then come back and remove the old OAI + the permissions for it

**Review by kscoulter — APPROVED** [review]

**Pull request merged**